ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-commands.sh

```diff
@@ -16,6 +16,23 @@ tar -czf - . | ssh "${SSHOPTS[@]}" "root@${IP}" "cat > /root/assisted.tar.gz"
 # Prepare configuration and run
 scp "${SSHOPTS[@]}" "${CLUSTER_PROFILE_DIR}/pull-secret" "root@${IP}:pull-secret"
 
+if [ "${ENVIRONMENT}" != "local" ]; then
+  if [ "${ENVIRONMENT}" = "production" ]; then
+    remote_service_url="https://api.openshift.com"
+    pull_secret_file=/var/run/assisted-installer-bot/prod-pull-secret
+  else
+    echo "Unknown environment ${ENVIRONMENT}"
+    exit 1
+  fi
+
+  scp "${SSHOPTS[@]}" /var/run/assisted-installer-bot/offline-token "root@${IP}:offline-token"
```

Comment:
Maybe put `/var/run/assisted-installer-bot` in a variable like `CI_CREDENTIALS_DIR` so it's a bit more clear where it's coming from

Reply:
done